src/Nest/Search/Search/Highlighting/HighlightField.cs

```diff
@@ -238,7 +241,7 @@ public class HighlightFieldDescriptor<T> : DescriptorBase<HighlightFieldDescript
 		public HighlightFieldDescriptor<T> AllField() => this.Field("_all");
 
 		/// <inheritdoc/>
-		public HighlightFieldDescriptor<T> TagsSchema(string schema = "styled") => Assign(a => a.TagsSchema = schema);
+		public HighlightFieldDescriptor<T> TagsSchema(HighlighterTagsSchema? schema) => Assign(a => a.TagsSchema = schema);
```

Comment:
No default?

Reply:
I can make it the default, but the client generally doesn't have default values for nullable enums. I suspect this had a default value more as a hint for the accepted string value.